command/server.go

```diff
@@ -1504,13 +1559,15 @@ CLUSTER_SYNTHESIS_COMPLETE:
 			if addr != "" {
 				tcpAddr, err := net.ResolveTCPAddr("tcp", lnConfig.ClusterAddress)
 				if err != nil {
+					observer.Error("listener-cluster-address", err)
 					c.UI.Error(fmt.Sprintf("Error resolving cluster_address: %s", err))
 					return 1
 				}
 				clusterAddrs = append(clusterAddrs, tcpAddr)
 			} else {
 				tcpAddr, ok := ln.Addr().(*net.TCPAddr)
 				if !ok {
+					observer.Error("listener-cluster-address", err)
```

Comment:
where is this `err` supposed to be coming from? i'm only seeing the err from 1544, and that's already handled?